cognitive/src/main/scala/com/microsoft/azure/synapse/ml/geospatial/AzureMapsHelpers.scala

```diff
@@ -19,11 +21,65 @@ import java.net.{URI, URLEncoder}
 import java.util.concurrent.TimeoutException
 import scala.concurrent.blocking
 import scala.language.{existentials, postfixOps}
-import spray.json.DefaultJsonProtocol.{StringJsonFormat, seqFormat}
+import spray.json.DefaultJsonProtocol.{DoubleJsonFormat, StringJsonFormat, seqFormat}
+
+trait HasSetGeography extends Wrappable with HasURL with HasUrlPath {
```

Comment:
Is this needed? Can we just use setLocation?

Reply:
Azure Maps creator is a geographic resource, although it does the same thing as `HasSetLocation`. This trait was created to reduce confusion